web_pwa_oca/controllers/main.py

```diff
@@ -1,11 +1,23 @@
+# Copyright 2020 Lorenzo Battistini @ TAKOBI
+# Copyright 2020 Tecnativa - Alexandre D. Díaz
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
 from odoo.http import request, Controller, route
 
 
 class PWA(Controller):
 
-    def get_asset_urls(self, asset_xml_id):
-        qweb = request.env['ir.qweb'].sudo()
-        assets = qweb._get_asset_nodes(asset_xml_id, {}, True, True)
+    def _get_pwa_scripts(self):
+        """ Scripts to be imported in the service worker (Order is important) """
```

Comment:
I don't understand where does this pattern of whitespacing docstring come from... :thinking: anyway: https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

```suggestion
        """Scripts to be imported in the service worker (Order is important)"""
```

Reply:
Docstrings are trimmed... so no real differences in the generated document. I like the blank space style because for me looks better.

And for me follows the single comment style:
```# My comment``` vs ```#My Commen```

Reply:
Visually it's not the same the glyph `#` that covers full height than `"` that only covers upper part. That's why in most conventions, you don't need to let an extra space between `"` and text, which is advised between `#` and text.

Reply:
I have no problem to change this, i will do that, but see this docstring:
```"""Get all url's that have 'asset_xml_id'"""``` vs ```""" Get all url's that have 'asset_xml_id' """```
or the extreme case:
```"""Get all url's that have \"asset_xml_id\""""``` vs ```""" Get all url's that have \"asset_xml_id\" """```

Reply:
But this is like black: the main thing is to be homogeneous, not to be pretty for all cases. The same would apply here.